AllReadyApp/Web-App/AllReady/ViewModels/Home/Index.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System.Collections.Generic;
-using AllReady.ViewModels.Campaign;
 
 namespace AllReady.ViewModels.Home
 {
-    public class HomePageViewModel
+    public class Index
```

Comment:
@mgmccarthy I was glancing through this at lunchtime. Curious on the naming choice for this class. While sometimes a bit redundant I tend to prefer the ViewModel suffix to make it pretty explicit what these classes are.